internals/cli/cli.go

```diff
@@ -30,6 +30,7 @@ import (
 	"golang.org/x/crypto/ssh/terminal"
 
 	"github.com/canonical/pebble/client"
+	"github.com/canonical/pebble/cmd"
```

Comment:
Yeah, perhaps. I don't love `version`, because then it's not at all clear it's the `cmd` package. If anything, I'd suggest `cmdpkg`. However, I'd suggest going the other way and change the receiver name to just `c`, for example `func (c *cmdEnter)`. Single-letter receiver names are the norm in Go, and there's a precedent for that in `cmd_changes.go` already.

Reply:
Let's stick with `c` (as a side note, I think `rcmd` comes from *run command*, which is something I think I saw in `cmd_run.go` and replicated here wrongly 🤡)